UPGRADING

```diff
@@ -130,6 +130,10 @@ PHP 8.5 UPGRADE NOTES
     RFC: https://wiki.php.net/rfc/marking_return_value_as_important
   . Added asymmetric visibility support for static properties.
     RFC: https://wiki.php.net/rfc/static-aviz
+  . Added support for attributes on compile-time non-class constants.
+  . Added constant Attribute::TARGET_CONSTANT.
+  . The #[\Deprecated] attribute can now be used on constants.
+    RFC: https://wiki.php.net/rfc/attributes-on-constants
```

Comment:
```suggestion
  . Added support for attributes on compile-time non-class constants.
    RFC: https://wiki.php.net/rfc/attributes-on-constants
  . The #[\Deprecated] attribute can now be used on constants.
    RFC: https://wiki.php.net/rfc/attributes-on-constants
```
The `Attribute::TARGET_CONSTANT` is implied and the RFC should be linked for every change, since it's not always clear what belongs together.